examples/next-forms/app/delete-form.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import { experimental_useFormState as useFormState } from 'react-dom'
+import { experimental_useFormStatus as useFormStatus } from 'react-dom'
+import { deleteTodo } from '@/app/actions'
+
+const initialState = {
+  message: null,
+}
+
+function DeleteButton() {
+  const { pending } = useFormStatus()
+
+  return (
+    <button type="submit" aria-disabled={pending}>
+      Delete
+    </button>
+  )
+}
+
+export function DeleteForm({ id }: { id: number }) {
+  const [state, formAction] = useFormState(deleteTodo, initialState)
+
+  return (
+    <form action={formAction}>
+      <input type="hidden" name="id" value={id} />
+      <DeleteButton {...state} />
```

Comment:
```suggestion
      <DeleteButton />
```

The `DeleteButton` component doesn't have any props defined above.